src/fancyline/tty/vt100.cr

```diff
@@ -1,5 +1,19 @@
 class Fancyline
   class Tty
+    lib LibC
+      struct Winsize
+        ws_row : UInt16    #  rows, in characters
+        ws_col : UInt16    #  columns, in characters
+        ws_xpixel : UInt16 #  horizontal size, pixels
+        ws_ypixel : UInt16 #  vertical size, pixels
+      end
+
+      IOC_OUT      = 0x40000000
+      IOCPARM_MASK =     0x1fff
+      TIOCGWINSZ   = IOC_OUT | ((sizeof(Winsize) & IOCPARM_MASK) << 16) | (('t'.ord) << 8) | 104
```

Comment:
I guess this is for Linux on `x86_64`?  Can you add documentation pointing this out?

On the long run one could use a `{% if flags(:x86_64) %}` construct to support different systems.

Reply:
I have absolutely no idea where to find out what to set these values for os' other than `x86_64`.

Reply:
Im positive it works on macOS

Reply:
Ah now I see, you're on Mac.  I assumed Linux, lul.  Still, the conditional should be there, although checking for `:darwin` instead of linux then

Reply:
The available platform flags aren't really documented.  You can look here for reference: https://github.com/Papierkorb/bindgen/blob/master/src/bindgen/variables.cr Those were all existing ones .. At least when I wrote that